src/layer/poi.js

```diff
@@ -0,0 +1,74 @@
+import * as label from "../constants/label.js";
+import * as Color from "../constants/color.js";
+
+export const poi = {
+  id: "poi",
+  type: "symbol",
+  paint: {
+    "text-halo-color": "#ffffff",
+    "icon-halo-color": "#ffffff",
+    "text-halo-width": 1.5,
+    "icon-halo-width": 0.4,
+    "text-halo-blur": 1,
+    "icon-halo-blur": 0.2,
```

Comment:
I think an opaque, land-colored knockout would feel more consistent than a blurry white halo, which also makes this non-bold, light-colored text pretty difficult to read.

Reply:
I came to the same conclusion and implemented this!